src/event-replay/parquet-based/importers/attachment-new-importer.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
+
+import { PgWriteStore } from '../../../datastore/pg-write-store';
+import { parseAttachmentMessage } from '../../../event-stream/event-server';
+import { logger } from '../../../logger';
+import { CoreNodeAttachmentMessage } from '../../../event-stream/core-node-message';
+import { DataStoreAttachmentSubdomainData, DataStoreBnsBlockData } from '../../../datastore/common';
+import { DatasetStore } from '../dataset/store';
+import { I32_MAX } from '../../../helpers';
+
+export const processAttachmentNewEvents = async (db: PgWriteStore, dataset: DatasetStore) => {
+  logger.info({ component: 'event-replay' }, 'ATTACHMENTS_NEW events process started');
+
+  const attachmentsNewEvents = await dataset.attachmentsNewEvents();
+  const ary: DataStoreAttachmentSubdomainData[] = [];
+
+  for await (const event of attachmentsNewEvents) {
+    const blockData: DataStoreBnsBlockData = {
+      index_block_hash: '',
+      parent_index_block_hash: '',
+      microblock_hash: '',
+      microblock_sequence: I32_MAX,
+      microblock_canonical: true,
+    };
+    const dataStore: DataStoreAttachmentSubdomainData = {};
```

Comment:
As we discussed, this is where things will get complicated... You should take a look at the current subdomain processing code [here](https://github.com/hirosystems/stacks-blockchain-api/blob/develop/src/datastore/pg-write-store.ts#L1387) to get an idea, but I'm worried that we won't be able to pull it off with the parquet-based replay.

Reply:
Thanks for linking the code! I'll double check the subdomain processing implementation.